independent-publisher-connectors/Financial Industry Regulatory Authority FINRA (Independent Publisher)/readme.md

```diff
@@ -0,0 +1,192 @@
+## Financial Regulatory Authority (FINRA) Connector
```

Comment:
Please follow the indentation as per the below link. 

https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/Independent%20Publisher/readme.md